runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/ContactSection/ContactForm/FormButtonGroup/FormButtonGroup.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC } from 'react';
+
+import Image from 'next/image';
+
+import RefreshImg from '#public/images/icons/refresh.svg';
+import LoaderImg from '#public/images/shapes/loader.svg';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+
+import { FormStatusType, Status } from '../ContactForm.types';
+import { FormButton } from '../FormFields';
+import styles from './FormButtonGroup.module.scss';
+
+const ResetBox: FC = () => (
+    <button className={styles.reset} type="reset">
+        <Image src={RefreshImg} width={24} height={24} alt=" " />
+    </button>
+);
+
+const AdditionalInfo: FC<{ status: Status }> = ({ status }) => {
+    if (status?.type === FormStatusType.LOADING) {
+        return <Image src={LoaderImg} width={50} height={50} alt=" " />;
+    }
+    if (status?.type === FormStatusType.SUCCESS) {
+        return <ResetBox />;
+    }
+    return <div />;
```

Comment:
return empty div? imo better use <If condition={} />

Reply:
im using grid in this container, so i need an element to keep grid working